.gitignore

```diff
@@ -1,5 +1,7 @@
 /out/
 /gen/
-src/com/zephir/lexer/ZephirLexer.java
 *~
 *.jar
+/Zephir.iml
+/Zephir.zip
+/.idea
```

Comment:
For these purposes create a global .gitignore file, which is a list of rules
for ignoring files in every Git repository on your computer.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore